data/data-files/server-side-eval/rollout-or-experiment.yml

```diff
@@ -0,0 +1,148 @@
+---
+name: rollout or experiment - <NAME>
```

Comment:
This file is the same as the equivalent one in https://github.com/launchdarkly/sdk-test-harness/pull/56 except that I changed the name "user" to "context". The actual context data here is still just simple users, because the point of the test cases here is not to test anything specifically involving the context attributes, but just the general behavior of iterating through the buckets and returning a variation.